testing/chainsaw/e2e/pgbackrest-restore/README.md

```diff
@@ -0,0 +1,5 @@
+# pgbackrest-restore
+
+This [chainsaw](https://github.com/kyverno/chainsaw) suite tests that CPK can clone and restore through pgBackRest backups.
+
+...other material here...
```

Comment:
What else should we put here to explain this to ourselves when we come back to this?

Reply:
I can't think of anything right now and the original test didn't have anything, so I'll leave this for now as a placeholder.